### ISSUE TITLE
Change New Post button accessibility label to Write

### ISSUE THREAD

**Issue by astralbodies** [wordpress-mobile/WordPress-Android#4815]
We are transitioning to "Write" as the verb for the new post button in the tab bar. Update the accessibility label accordingly. No plans yet to visually change the UI based upon this label on larger screens as of yet.

Related iOS issue: https://github.com/wordpress-mobile/WordPress-iOS/issues/6222

**Comment by astralbodies** [comment]
I just realized that the Android app doesn't have the New Post button in the tab bar like on iOS. But I think updating the label for the orange compose button makes sense.

**Comment by astralbodies** [comment]
https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/values/strings.xml#L967

**Comment by hypest** [comment]
There is no textual label on the FAB (the orange compose button) or a textual hint either so, I don't thing there is something to be worked on there for the moment.

> https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/values/strings.xml#L967

Aha, so, that string is only used when the notifications list is empty and we prompt the user to create a new post. Besides, it's "new post" currently, which is not the verbal form and I think we don't really need to change it into "write a post".

In any case, I'd recommend we remove the [6.2](https://github.com/wordpress-mobile/WordPress-Android/milestone/74) milestone from this one. WDYT @maxme ?

**Comment by maxme** [comment]
> There is no textual label on the FAB (the orange compose button) or a textual hint either so, I don't thing there is something to be worked on there for the moment.

It's used [here](https://github.com/wordpress-mobile/WordPress-Android/blob/242aa95fcd64bd441e21d387b9275c0d592e057d/WordPress/src/main/res/layout/my_site_fragment.xml#L464-L464), this text will be read when the app is used with TalkBack.

> In any case, I'd recommend we remove the 6.2 milestone from this one. WDYT @maxme ?

It's not urgent, can wait the next code freeze.

**Comment by hypest** [comment]
> It's used here, this text will be read when the app is used with TalkBack.

Hmm, this is the second time recently that searching through the app via AS has lead me to poor results! Not fun. Will pay some more attention to it. Thanks @maxme !
